cds/modules/previewer/templates/cds_previewer/video.html

```diff
@@ -97,10 +97,15 @@
       {% endif %}
       {% if log_media_views %}
         (function(player) {
+          var url = '{{ (config.LOG_USER_ACTIONS_ENDPOINTS['media_view'] or '')
+                                      .format(base_url=config.LOG_USER_ACTIONS_ENDPOINTS['base_url'] or '',
```

Comment:
the syntax  is a bit weird , IMHO should have a syntax error on JS WDYT? with ``' ... or '' ''`` and the multi lines

Reply:
Jinja2 will replace everything before the browser will receive the page, so for JS it will be just something like:
`var url = 'domain.com/etc';`